frontend/benefit/handler/src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,22 @@
+import { useTranslation } from 'benefit/handler/i18n';
```

Comment:
This could be in shared folder if importing straight from `import { useTranslation } from 'from 'next-i18next';` Kesaseteli could also use it, every service just then implement their own translation texts.

Reply:
fixed